packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +926,47 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting
+   * @param joinMeetingParams
+   */
+  export function joinMeeting(joinMeetingParams: JoinMeetingParams): Promise<string | object> {
+    return new Promise<string | object>((resolve) => {
+      ensureInitialized(runtime, FrameContexts.content);
+      resolve(
+        sendAndHandleStatusAndReasonWithVersion(
+          getApiVersionTag(ApiVersionNumber.V_2, ApiName.Meeting_JoinMeeting),
+          'meeting.joinMeeting',
+          joinMeetingParams,
+        ),
+      );
+    });
+  }
+
+  /** Join meeting parameters */
+  export interface JoinMeetingParams {
+    /** The meeting url to join. */
+    skypeTeamsMeetingUrl: string;
+    /** The subject of the meeting. */
+    subject: string;
+    /** The conversation id of the meeting. */
+    source: EventActionSource;
+    /** The conversation id of the meeting. */
+    conversationId: string;
+    /** The message id of the meeting. */
+    messageId: string;
+    /** The root message id of the meeting. */
+    rootMessageId: string;
+  }
+
+  /** Event action source */
+  export type EventActionSource =
```

Comment:
This would be easier for developers to use as an `enum` (see the `MeetingReactionType` enum in this file for an example)

Reply:
Changed it to enum